ci/actions-templates/linux-builds-template.yaml

```diff
@@ -19,24 +19,25 @@ jobs: # skip-master skip-pr skip-stable
           - x86_64-unknown-linux-gnu
           - armv7-unknown-linux-gnueabihf
           - aarch64-linux-android
-          - aarch64-unknown-linux-gnu     # skip-pr
-          - aarch64-unknown-linux-musl    # skip-pr skip-master
-          - powerpc64-unknown-linux-gnu   # skip-pr
-          - x86_64-unknown-linux-musl     # skip-pr
-          - i686-unknown-linux-gnu        # skip-pr skip-master
-          - arm-unknown-linux-gnueabi     # skip-pr skip-master
-          - arm-unknown-linux-gnueabihf   # skip-pr skip-master
-          - x86_64-unknown-freebsd        # skip-pr skip-master
-          - x86_64-unknown-netbsd         # skip-pr skip-master
-          - x86_64-unknown-illumos        # skip-pr skip-master
-          - powerpc-unknown-linux-gnu     # skip-pr skip-master
-          - powerpc64le-unknown-linux-gnu # skip-pr skip-master
-          - s390x-unknown-linux-gnu       # skip-pr skip-master
-          - arm-linux-androideabi         # skip-pr skip-master
-          - armv7-linux-androideabi       # skip-pr skip-master
-          - x86_64-linux-android          # skip-pr skip-master
-          - riscv64gc-unknown-linux-gnu   # skip-pr skip-master
-          - loongarch64-unknown-linux-gnu # skip-pr skip-master
+          - aarch64-unknown-linux-gnu      # skip-pr
+          - aarch64-unknown-linux-musl     # skip-pr skip-master
+          - powerpc64-unknown-linux-gnu    # skip-pr
+          - x86_64-unknown-linux-musl      # skip-pr
+          - i686-unknown-linux-gnu         # skip-pr skip-master
+          - arm-unknown-linux-gnueabi      # skip-pr skip-master
+          - arm-unknown-linux-gnueabihf    # skip-pr skip-master
+          - x86_64-unknown-freebsd         # skip-pr skip-master
+          - x86_64-unknown-netbsd          # skip-pr skip-master
+          - x86_64-unknown-illumos         # skip-pr skip-master
+          - powerpc-unknown-linux-gnu      # skip-pr skip-master
+          - powerpc64le-unknown-linux-gnu  # skip-pr skip-master
+          - s390x-unknown-linux-gnu        # skip-pr skip-master
+          - arm-linux-androideabi          # skip-pr skip-master
+          - armv7-linux-androideabi        # skip-pr skip-master
+          - x86_64-linux-android           # skip-pr skip-master
+          - riscv64gc-unknown-linux-gnu    # skip-pr skip-master
+          - loongarch64-unknown-linux-gnu  # skip-pr skip-master
+          - loongarch64-unknown-linux-musl # skip-pr skip-master skip-stable
```

Comment:
Hmmm so you're saying that the CI should completely ignore this target as for now, if I'm not mistaken?

Reply:
Yes. If I'm not mistaken, rustup is currently only built with stable rustc. The first stable version of `loongarch64-linux-musl` will be 1.81, when `skip-stable` will be removed. Thanks

Reply:
@heiher I see. Thanks for the explanation!

Reply:
@heiher Are we ready for releasing this on `stable` now? This should be the right time if possible since we're releasing v1.28.0 very soon:
- #3998

Reply:
@heiher (a gentle ping...)

Reply:
Sorry, I missed the previous notification. I’ll enable the stable build for this target soon. Thanks for your mention.

Reply:
@heiher Thanks for your approval! I'll place this directly in https://github.com/rust-lang/rustup/pull/4105 for the sake of convenience :)